src/components/Common/RecordListFilter.vue

```diff
@@ -401,7 +401,7 @@ export default {
 }
 
 .filter-field-editor {
-  width: -webkit-fill-available;
+  width: strech;
```

Comment:
Are you sure this is correct ?

Reply:
yes, it's working like before. There's a warning which appears if this isn't changed